### PR TITLE
support stdin

### PR DESCRIPTION
**Pull request by daisuzu**
This change makes to be able to use from vim by `:%!gofmtmd`.

### CODE REVIEW

File: cmd/gofmtmd/main.go [hunk 1]
```diff
@@ -19,10 +19,19 @@ var rootCmd = &cobra.Command{
 	Use:     "gofmtmd",
 	Version: "0.1.1",
 	Short:   "This CLI formats Go Code in Markdown.",
-	Args:    cobra.MinimumNArgs(1),
+	Args:    cobra.MinimumNArgs(0),
```
Comment (po3rin) [inline cmd/gofmtmd/main.go]: If you want to receive less than one argument, I think we should use ```MaximumNArgs```.

https://github.com/spf13/cobra/blob/HEAD/args.go#L62

```diff
- Args:    cobra.MinimumNArgs(0),
+ Args:    cobra.MaximumNArgs(1),
```

**Review by po3rin — COMMENTED** [review]
Thanks for PR！I think that there are some points to discuss.

File: cmd/gofmtmd/main.go [hunk 2]
```diff
@@ -19,10 +19,19 @@ var rootCmd = &cobra.Command{
...
+		if len(args) > 0 {
+			filename = args[0]
+		}
+		var md []byte
+		var err error
+		if filename != "" {
+			md, err = ioutil.ReadFile(filename)
+		} else {
+			md, err = ioutil.ReadAll(os.Stdin)
+		}
```
Comment (po3rin) [inline cmd/gofmtmd/main.go]: Just decide what to use as io.Reader, how about using ioutil.ReadAll in common? I think this is useful to support other io.Readers in the future

```go
// omit error handling ...

reader := os.Stdin
if filename != "" {
	reader, _ = os.Open(filename) 
} 

md, _ := ioutil.ReadAll(reader)
```

File: cmd/gofmtmd/main.go (hunk 1 above)
Comment (daisuzu) [reply cmd/gofmtmd/main.go]: Thank you!
I didn't know `MaximumNArgs`

**Review by po3rin — APPROVED** [review]
Thanks for  your PR !!

**Comment by daisuzu** [comment]
Thanks!

**Pull request merged**